src/_pytest/assertion/rewrite.py

```diff
@@ -949,11 +949,19 @@ def visit_BinOp(self, binop):
         res = self.assign(ast.BinOp(left_expr, binop.op, right_expr))
         return res, explanation
 
+    def _is_any_call_with_generator_or_list_comprehension(self, call):
```

Comment:
can be `@staticmethod` (does not use `self`)

Reply:
Done! 👍